### ISSUE TITLE
Does `LangevinIntegrator.setTemperature()` update the temperature of the context?

### ISSUE THREAD

**Issue by zhang-ivy** [openmm/openmm#3407]
If I call `integrator.setTemperature()` (where `integrator` is a `LangevinMiddleIntegrator` or `LangevinIntegrator`) after creating a context, does the context temperature update? Or is the context temperature fixed after creation?

**Comment by peastman** [comment]
Yes, it gets updated.  An Integrator is bound to a particular Context, so it functions as part of the Context (which can change at any time) rather than as part of the System (which can be shared between many Contexts).

**Comment by jchodera** [comment]
Thanks for labeling this as `documentation`! It would be great if the API docs for the integrators clarified which parameters could be updated for an active `Context` (or, of all of them can be updated, note so at the top of the API docs for each class).
